packages/components/src/flex/stories/index.tsx

```diff
@@ -0,0 +1,80 @@
+/**
+ * External dependencies
+ */
+import type { ComponentMeta, ComponentStory } from '@storybook/react';
+
+/**
+ * Internal dependencies
+ */
+import { Flex, FlexItem, FlexBlock } from '../';
+import { View } from '../../view';
+
+const meta: ComponentMeta< typeof Flex > = {
+	component: Flex,
+	title: 'Components/Flex',
+	argTypes: {
+		align: { control: { type: 'text' } },
```

Comment:
For the flex options that require "valid" CSS values (like `align` and `justify`), should we show a dropdown the available correct values instead of a text field?

Reply:
Sounds like a good idea to me. Do you know if there a way to get these values automatically or is it a manual job?

Reply:
I'm afraid it's a manual job, unless we can just remove the `type: 'text'` override and hope that Storybook's docgen will display the available options correctly?

Reply:
Personally I don't think it's worth it if we have to populate a list manually.

Reply:
When removing the override the story lets you `Set object`, so I think it's better to leave it as is.

Reply:
Cool! Thank you for checking anyway, it was worth trying.